"Operations" extensions not working on main

### Describe the Bug

I'm running my Directus from main bc I need some fixes merged after the last release. 

I have `operations` extensions that were working fine and are now broken. Problem also exists when testing with a sample `operations` crafted by Directus CLI according to the documentation. 

`[23:39:52.394] WARN: Couldn't register operation "test"
[23:39:52.394] WARN: Cannot find module '/extensions/operations/test/api.js' imported from /directus/node_modules/.pnpm/file+api/node_modules/@directus/api/dist/extensions/manager.js
    err: {
      "type": "Error",
      "message": "Cannot find module '/extensions/operations/test/api.js' imported from /directus/node_modules/.pnpm/file+api/node_modules/@directus/api/dist/extensions/manager.js",
      "stack":
          Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/extensions/operations/test/api.js' imported from /directus/node_modules/.pnpm/file+api/node_modules/@directus/api/dist/extensions/manager.js
              at new NodeError (node:internal/errors:405:5)
              at finalizeResolution (node:internal/modules/esm/resolve:327:11)
              at moduleResolve (node:internal/modules/esm/resolve:946:10)
              at defaultResolve (node:internal/modules/esm/resolve:1132:11)
              at nextResolve (node:internal/modules/esm/loader:163:28)
              at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
              at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
              at ESMLoader.import (node:internal/modules/esm/loader:524:22)
              at importModuleDynamically (node:internal/modules/esm/translators:110:35)
              at importModuleDynamicallyCallback (node:internal/process/esm_loader:36:14)
      "code": "ERR_MODULE_NOT_FOUND"
    }`


My `hooks` are working fine. I didn't test any other kind of extensions (only `operations` and `hooks`). 

I looked to the git history and saw some changes made by @rijkvanzanten on [this commit ](https://github.com/directus/directus/commit/1383e85fe93e0a58bcc249b2988ca34ec4aaa750) regarding extensions, I'm not sure if it is the one that broke it but building from [the one right before](https://github.com/directus/directus/commit/8dc68730ffe4f57a6fdce6bfd5d28731cee45060) is working well. 

### To Reproduce

Use Directus from main. 

Craft & build `operations` extensions as described on [the documentation](https://docs.directus.io/extensions/operations.html). 

Launch Directus, error message should be printed. 

### Directus Version

main

### Hosting Strategy

Self-Hosted (Docker Image)

--- a/.changeset/polite-chefs-run.md
+++ b/.changeset/polite-chefs-run.md
@@ -1,0 +1,5 @@
+---
+"@directus/api": patch
+---
+
+Fixed incorrect extension loading path

--- a/api/src/extensions/manager.ts
+++ b/api/src/extensions/manager.ts
@@ -384,7 +384,7 @@ export class ExtensionManager {
 				const operationPath = path.resolve(operation.path, operation.entrypoint.api!);
 
 				const operationInstance: OperationApiConfig | { default: OperationApiConfig } = await import(
-					`../${pathToRelativeUrl(operationPath, __dirname)}?t=${Date.now()}`
+					`./${pathToRelativeUrl(operationPath, __dirname)}?t=${Date.now()}`
 				);
 
 				const config = getModuleDefault(operationInstance);
@@ -407,7 +407,7 @@ export class ExtensionManager {
 				const bundlePath = path.resolve(bundle.path, bundle.entrypoint.api);
 
 				const bundleInstances: BundleConfig | { default: BundleConfig } = await import(
-					`../${pathToRelativeUrl(bundlePath, __dirname)}?t=${Date.now()}`
+					`./${pathToRelativeUrl(bundlePath, __dirname)}?t=${Date.now()}`
 				);
 
 				const configs = getModuleDefault(bundleInstances);

